src/notebooks/R Mia Examples/Comparative-Metagenomics.rmarkdown

```diff
@@ -0,0 +1,1118 @@
+                          assay.type = "relabundance",
+                          rank = "Class",
+                          order_rank_by = "abund")
+
+# Plotting our rarefied data
+rarefy_plot <- plotAbundance(rarefy_tse,
+                             assay.type = "rarefy",
+                             rank = "Class",
+                             order_rank_by = "abund")
+```
+
+
+
+
+By naked eye, we can see that the profiles change. Let’s take a deeper look at 
+some interesting Classes to check how the abundance change depending on the 
+normalization method.
+
+1. Define our subset classes
+
+
+
+
+```{r}
+subset_classes <- c("Betaproteobacteria", 
+                    "Bacteroidia", 
+                    "Actinobacteria")
+```
+
+
+
+
+
+2. Subset our `TreeSummarizedExperiment` objects.
+
+
+
+
+```{r}
+class_subset <- filtered_tse[rowData(filtered_tse)$Class %in% subset_classes, ]
+```
+
+
+
+
+3. Plot our new plots with the `plotAbundance` `mia` method.
+
+
+
+
+```{r}
+tss_plot <- plotAbundance(class_subset, 
+                          assay.type = "relabundance", 
+                          rank = "Class", 
+                          order_rank_by = "abund")
+
+rarefy_plot <- plotAbundance(class_subset, 
+                             assay.type = "rarefy", 
+                             rank = "Class", 
+                             order_rank_by = "abund")
+```
+
+
+
+
+
+## Comparative metagenomics at community level: Beta diversity
+
+Beta (β)-diversity is used in ecology to assess the heterogeneity of local communities. 
+It is defined as the ratio between regional and local species diversity. In simpler 
+terms, it calculates the number of species that are not the same in two different 
+environments.
+
+Some of the most popular beta diversity measures in microbiome research include 
+Bray-Curtis index (for compositional data), Jaccard index (for presence/absence 
+data, ignoring abundance information), Aitchison distance (Euclidean distance 
+for transformed abundances, aiming to avoid the compositionality bias), and the 
+Unifrac distances (that use the phylogenetic tree information). 
+
+Only some of the commonly used beta diversity measures are actual distances, 
+therefore, the term dissimilarity or beta-diversity is commonly used. The workflow 
+to assess β-diversity between groups of samples starts with calculating the diversity 
+indices for each pair of samples. This results in a distance (or dissimilarity) 
+matrix that is often used for ordination (e.g., principal coordinates analysis) 
+and data exploration in microbiota data analysis ([6](#ref-wagner2018)).
+
+For statistical analysis, [adonis](https://www.rdocumentation.org/packages/vegan/versions/2.6-4/topics/adonis) 
+and [betadisper](https://www.rdocumentation.org/packages/vegan/versions/2.6-4/topics/betadisper) 
+methods are widely used together in β-diversity analyses.
+
+Adonis tests whether two or more groups have different compositions. Adonis 
```

Comment:
The method is PERMANOVA, and adonis is the vegan function that can be used to calculate that (plus some other things)

Reply:
-> I suggest to use name PERMANOVA here instead of Adonis